src/Components/Pages/Plot/JSRootGraph.js

```diff
@@ -1,17 +1,58 @@
-import React, { createRef } from "react";
+import React, { createRef } from 'react';
+import { StoppingPowerUnits } from '../../../Backend/Utils';
 
-let JSROOT
-//#region Helper functions
+function power10Max(values) {
+    return Math.pow(10, Math.ceil(Math.log10(Math.max(...values))))
+}
+
+function power10Min(values) {
+    return Math.pow(10, Math.floor(Math.log10(Math.min(...values))))
+}
+
+function checkBoundaries(dataSeries) {
+    if(dataSeries.length < 1) return {
+        minEnergy: 0,
+        maxEnergy: 10,
+        minValue: 0,
+        maxValue: 10,
+    }
+    // We're not handling such big energies so it's safe to assume everything will be smaller than 1e+10
+    const boundaries = {
+        minEnergy: 1e+10,
```

Comment:
As for reasonable borders for energy we can assume that currently we have some knowledge until `10^8` MeV/nucl

Reply:
Same answer as to the comment above